lib/_emerge/Scheduler.py

```diff
@@ -1193,6 +1199,10 @@ def sighandler(signum, frame):
                     signal.signal(signal.SIGCONT, earlier_sigcont_handler)
                 else:
                     signal.signal(signal.SIGCONT, signal.SIG_DFL)
+                if earlier_sigusr2_handler is not None:
```

Comment:
No need to do this now, but I wonder if we could do bette rwith this with a context manager.